__tests__/app.test.js

```diff
@@ -65,10 +65,42 @@ describe('Getting a list of available api', ()=> {
   })
 
 
+  describe('test to see if input id of article exists', ()=> {
+    test('to see if object is returned', ()=> { 
+    return request(app)
+   .get(`/api/articles/2`)
+    .expect(200)
+    .then(({body}) => {
+
+
+  
+    
+
+    expect(body).toHaveProperty("article_id", expect.any(Number))
```

Comment:
expect.any(number) definitely works here however since we're asking for a specific article we can be a little more explicit with what id we'd expect back.